ArduCopter/GCS_Mavlink.cpp

```diff
@@ -724,6 +724,15 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_long_packet(const mavlink_command_
             return MAV_RESULT_FAILED;
         }
         return MAV_RESULT_ACCEPTED;
+#if MODE_GUIDED_ENABLED == ENABLED
+    case MAV_CMD_NAV_LOITER_TURNS:
+        if(copter.get_mode() == (uint8_t)Mode::Number::GUIDED)
```

Comment:
```suggestion
        if (copter.flightmode->in_guided_mode()) {
```

Reply:
@khancyr  sir, I have made the changes. Please take a look!